jetty-runtime/jetty9-base/pom.xml

```diff
@@ -122,7 +135,7 @@
           <arguments>
             <argument>-jar</argument>
             <argument>../jetty-distribution-${jetty.version}/start.jar</argument>
-            <argument>--add-to-startd=gae,http,jsp,jstl</argument>
+            <argument>--add-to-startd=http,deploy,jsp,jstl,http-forwarded,resources,gae</argument>
```

Comment:
Note, the modifications to `gae.mod` (further down in this commit) makes the addition of `deploy` and `resources` on this `<argument>` irrelevant, and those 2 modules can be safely removed from this line.